app/Http/Controllers/ApiController.php

```diff
@@ -129,6 +144,56 @@ public function getpaymentstatus($gateway, $transactionId, $payToken, $paymentRe
             "pay_ref" => $paymentRef
 
         ]);
-        return $response->json();
-    }
+
+        
+        return $response->json();    }
+
+
+
+
+        public function transactionStatus()
```

Comment:
Getting the transaction status should have the same convention as mentioned in the make payment implementation